src/Resources/app/administration/src/module/sw-in-app-purchases/component/sw-in-app-purchase-checkout-subscription-change/index.ts

```diff
@@ -0,0 +1,69 @@
+import type * as IAP from "SwagExtensionStore/module/sw-in-app-purchases/types";
+import template from "./sw-in-app-purchase-checkout-subscription-change.html.twig";
+import './sw-in-app-purchase-checkout-subscription-change.scss';
+
+export default Shopware.Component.wrapComponentConfig({
+    template,
+
+    props: {
+        purchase: {
+            type: Object as PropType<IAP.InAppPurchase>,
+            required: true,
+        },
+        cart: {
+            type: Object as PropType<IAP.InAppPurchaseCart>,
+            required: true,
+        },
+    },
+
+    computed: {
+        locale() {
+            const local = String(Shopware.Store.get('session').currentLocale ??
+                Shopware.Store.get('context').app?.fallbackLocale ?? 'en-US');
```

Comment:
`en-US` is not the real "default". Coming from language plugin, right?

```suggestion
                Shopware.Store.get('context').app?.fallbackLocale ?? 'en-GB');
```